<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>751</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>271</width>
       <height>731</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="ContactList"/>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>891</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextBrowser" name="MessageDisplay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>871</width>
       <height>591</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>630</y>
      <width>901</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="MessageToSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>781</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Send">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>70</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black; </string>
     </property>
     <property name="text">
      <string>Отправить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="emoji">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>2</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black; </string>
     </property>
     <property name="text">
      <string>Отправить</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1222</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Профиль</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Поменять пароль</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
